### PR TITLE
chore: add installation method in template

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Add how we installed Podman Desktop for bug report

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

N/A

### How to test this PR?

N/A

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -25,10 +25,17 @@ body:
     attributes:
       label: Operating system
       description: "Which operating system are you on? Please provide the version as well. If you are on a Mac, please specify Apple silicon or Intel."
-      placeholder: "macOS Monterey 12.2.1 (Intel)"
+      placeholder: "macOS Ventura 13.3 (Arm), Windows 11"
     validations:
       required: true 
 
+  - type: input
```
Comment (jeffmaury) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: Can't it be a dropdown ?

**Review by jeffmaury — APPROVED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (benoitf) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: it can, but I'm not sure we can list all the install methods ?

Comment (jeffmaury) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Yes we can have an extra option Other

Comment (benoitf) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: yes but how do you fill the field for 'other' ? it means to add an extra field with user input ?

Comment (jeffmaury) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: I would say let's start without an extra field and watch if Other is widely used or not

**Review by gbraad — APPROVED** [review]

**Review by deboer-tim — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (benoitf) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: ok fixed 👍

**Pull request merged**